apps/web/src/assets/icons/index.ts

```diff
@@ -1,3 +1,16 @@
-export { LogoDarkIcon } from "./LogoDark";
-export { LogoLightIcon } from "./LogoLight";
-export { MenuIcon } from "./Menu";
+export { default as LogoDarkIcon } from "./logo-dark.svg?react";
+export { default as LogoLightIcon } from "./logo-light.svg?react";
+export { default as MenuIcon } from "./menu.svg?react";
+export { default as ChevronDownIcon } from "./chevron-down.svg?react";
+export { default as FileCopyIcon } from "./file-copy.svg?react";
+export { default as PlusIcon } from "./plus.svg?react";
+export { default as ArrowDownLeftIcon } from "./arrow-down-left.svg?react";
+export { default as GoogleSocialIcon } from "./google-social.svg?react";
+export { default as FacebookSocialIcon } from "./facebook-social.svg?react";
+export { default as XSocialIcon } from "./x-social.svg?react";
+export { default as UnknownSocialIcon } from "./unknown-social.svg?react";
+export { default as LedgerSocial } from "./ledger-social.svg?react";
+export { default as MailIcon } from "./mail.svg?react";
```

Comment:
are those imported? I cannot find the files

Reply:
They are not in use yet, but ready to be